core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -69,4 +69,12 @@ public boolean isConnectedToMobileNetwork() {
         return networkInfo != null && networkInfo.isConnected();
     }
 
+    public String getMobileNetworkSubtypeName() {
+        NetworkInfo networkInfo = getNetworkInfo();
+        if (networkInfo == null || !networkInfo.isConnected()) {
+            return "";
```

Comment:
I've recently been avoiding extracting string constants unless it helps meaning/avoids duplication :scream:

In this case, I felt it might be more useful to know that you're explicitly sending an empty string back, but there's no point having a constant called `EMPTY_STRING`